pom.xml

```diff
@@ -840,6 +839,9 @@
                   <message>No SNAPSHOT dependencies are allowed!</message>
                   <onlyWhenRelease>true</onlyWhenRelease>
                 </requireReleaseDeps>
+                <requireMavenVersion>
+                  <version>3.2.5</version>
```

Comment:
If we're gonna enforce this at all might want to enforce 3.8.1 (https://maven.apache.org/docs/3.8.1/release-notes.html) because that has some supply chain CVE related fixes

Reply:
3.2.5 was just the number the versions plugins referred to.

That's a good reason to go to 3.8.1 (released 2021-03-30).

Done.